quipucords/api/migrations/0040_merge_20230926_2130.py

```diff
@@ -0,0 +1,12 @@
+# Generated by Django 4.2.4 on 2023-09-26 21:30
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("api", "0036_alter_credential_cred_type_alter_source_source_type"),
+        ("api", "0039_remove_scantask_details_report"),
```

Comment:
ah, now I see you did a merge migration... pls don't, we don't need branching in our migrations.